mco/machine-config-server.deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: machine-config-server1
```

Comment:
why are all of these names suffixed with "1"?

Reply:
just to separate from the one that runs as a daemon set.  We can name it whatever.

For the most part, probably don't want to use this deployment yet.  Stuff in ignition-generate is what you want for now.